build.sbt

```diff
@@ -8,6 +8,8 @@ ThisBuild / organization := "com.raphtory"
 ThisBuild / organizationName := "raphtory"
 ThisBuild / organizationHomepage := Some(url("https://raphtory.readthedocs.io/"))
 
+ThisBuild / resolvers += "Mulesoft Repository" at "https://repository.mulesoft.org/nexus/content/repositories/public/"
```

Comment:
Can you add a a comment to indicate why this is required. E.g., the StackOverflow thread.